<launch>
  <env name="ROSCONSOLE_FORMAT" value="[${severity}][${thread}][${node}/${function}:${line}]: ${message}"/>

  <node pkg="imu_bridge" type="imu_bridge" name="monitor" output="screen" respawn="true">
    <param name="serial_dev" value="/dev/serial/by-id/usb-Teensyduino_USB_Serial_5440650-if00" />
    <param name="baudrate" type="int" value="115200"/>

    <param name="battery_topic" value="/battery" />

    <param name="tf2_base_link" value="base_footprint" />
    <param name="tf2_output" value="base_link" />
    <param name="tf2_output_stabilized" value="base_stabilized" />
  </node>

</launch>
